multimodal/tests/unittests/test_distiller.py

```diff
@@ -86,3 +86,32 @@ def test_distillation():
         save_path=student_save_path,
     )
     verify_predictor_save_load(predictor, dataset.test_df)
+
+    # test for distillation with different model structures
+    different_hyperparameters = {
+        "optimization.max_epochs": 1,
+        "model.names": ["hf_text", "timm_image", "fusion_mlp"],
+        "model.hf_text.checkpoint_name": "google/electra-small-discriminator",
+        "model.timm_image.checkpoint_name": "swin_tiny_patch4_window7_224",
+        "env.num_workers": 0,
+        "env.num_workers_evaluation": 0,
```

Comment:
We may include the hparams about distillation here.

Reply:
Done